demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -139,8 +151,17 @@ def __len__(self):
 
 
 def get_model(ie, args):
+    if args.architecture_type in ('ctpn', 'yolo', 'yolov4', 'retinaface') and \
+       (
+           args.reverse_input_channels or
+           not np.array_equal(args.mean_values, [0., 0., 0.]) or
+           not np.array_equal(args.scale_values, [1., 1., 1.])
+       ):
```

Comment:
This also seems like it should be moved to `InputTransform` (e.g. `input_transform.is_trivial`).